playbooks/02 - Use Case - OTbase Inventory/OTbase Inventory > Link ICS Advisory to Assets.json

```diff
@@ -0,0 +1,284 @@
+            "stepType": "\/api\/3\/workflow_step_types\/12254cf5-5db7-4b1a-8cb1-3af081924b28",
+            "group": null,
+            "uuid": "e25bfde0-e5a8-4bfb-b2ae-1122096d1f5d"
+        },
+        {
+            "@type": "WorkflowStep",
+            "name": "Configuration",
+            "description": null,
+            "arguments": [],
+            "status": null,
+            "top": "165",
+            "left": "300",
+            "stepType": "\/api\/3\/workflow_step_types\/04d0cf46-b6a8-42c4-8683-60a7eaa69e8f",
+            "group": null,
+            "uuid": "1e3ca470-0d32-4e2b-9721-81b797e34e57"
+        },
+        {
+            "@type": "WorkflowStep",
+            "name": "Fetch ICS Advisories",
+            "description": null,
+            "arguments": {
+                "query": {
+                    "sort": [],
+                    "limit": 30,
+                    "logic": "AND",
+                    "filters": [
+                        {
+                            "type": "primitive",
+                            "field": "vendor",
+                            "value": "%{{vars.input.records[0].vendor}}%",
+                            "operator": "like",
+                            "_operator": "like"
+                        }
+                    ]
+                },
+                "module": "i_c_s_advisories?$limit=300",
+                "step_variables": []
+            },
+            "status": null,
+            "top": "300",
+            "left": "300",
+            "stepType": "\/api\/3\/workflow_step_types\/b593663d-7d13-40ce-a3a3-96dece928770",
+            "group": null,
+            "uuid": "059d1662-36af-4cae-ba7a-9391004326bf"
+        },
+        {
+            "@type": "WorkflowStep",
+            "name": "Fuzzy Search using Product",
+            "description": null,
+            "arguments": {
+                "name": "Fuzzy Search",
+                "params": {
+                    "cutoff": 75,
+                    "inputJSON": "{{vars.steps.Fetch_ICS_Advisories}}",
+                    "fuzzyMatch": true,
+                    "targetKeyValue": "{'product':\"{{vars.input.records[0].product}}\"}",
+                    "similarityThreshold": 75
+                },
+                "version": "1.0.0",
+                "connector": "fuzzy-search",
+                "operation": "filter_json_by_key_value",
+                "operationTitle": "Filter JSON by Key Value",
+                "pickFromTenant": false,
+                "step_variables": []
+            },
+            "status": null,
+            "top": "435",
+            "left": "300",
+            "stepType": "\/api\/3\/workflow_step_types\/0bfed618-0316-11e7-93ae-92361f002671",
+            "group": null,
+            "uuid": "a832ee7a-da54-49a8-b3fa-b4a8a49b54d2"
+        },
+        {
+            "@type": "WorkflowStep",
+            "name": "Link Asset and ICS Advisories",
+            "description": null,
+            "arguments": {
```

Comment:
Instead of looping through the "Link Asset and ICS Advisories" step, create a list of ICS advisory IRIs and link all of them to the asset in one go